scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/InputSizeReducerEstimator.scala

```diff
@@ -32,8 +32,8 @@ object InputSizeReducerEstimator {
     Common.inputSizes(info.step) match {
       case Nil =>
         LOG.warn("InputSizeReducerEstimator unable to estimate reducers; " +
-          "cannot compute size of:\n - " +
-          Common.unrollTaps(info.step).filterNot(_.isInstanceOf[Hfs]).mkString("\n - "))
+          "cannot compute size one of:\n - " +
```

Comment:
I'm ok with leaving the API as-is for now. Maybe just update the text to say that we typically can't compute sizes of memory taps / files not found. That will make debugging in the future easier.